packages/web-components/src/components/cbp-list/cbp-list.stories.tsx

```diff
@@ -5,7 +5,7 @@ export default {
         // variant: {
         //   control: 'select',
         //   description: 'Variant of List',
-        //   options: ['simple' , 'link-internal' , 'link-external'],
+        //   options: ['unstyled' , 'link-internal' , 'link-external'],
```

Comment:
why do we need separate variants for link-internal and link-external? I don't think we do.